### PR TITLE
Feature/client

### PR DESCRIPTION
**Pull request by mr-mig**

### CODE REVIEW

File: js/client.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ define([
 
 	var INBOX = 'http://inbox.yetudev.com/publish';
 	var OUTBOX = 'http://outbox.yetudev.com';
+	//var INBOX = 'http://localhost:9000/publish';
```
Comment (azadorozhniy) [inline js/client.js]: Please delete dead code.

File: test/client.spec.js [hunk 2]
```diff
@@ -6,23 +6,11 @@ define('socket.io-client', [], function () {
...
-	'bNoW6jpl_fBloP-ZfxJuppIhLuRyxE6jVOc57x8LOGhNtaWaQmvvK8GnHZa1aIOKt' +
-	'yHt1Qm263ZbNRgDKOCPfZOIvrNU_g7-aCYQik8oaFCiD2aTA7qK-VXvmYchY9tgAq' +
-	'2wGZ2qjmIdYczbuzDdCH-1lrYW9JGeCKA_gJnMrWQRbMrlHZLj0obl0GyHO30M55A' +
-	'4CGLo1VTd-NG2YbxhnEoO8LEOw4XvzSn5ISBY1ht5A7mGAx8mdh79Wz1W0WY7N-ZH' +
-	'RsX5G9uowR1GVaBtxpiykwE5yDHjmF3FzlJZB3WHjBKHi0I6Vqzl_Wt2EAK72gnlw' +
-	'PPag9eUgyub0Jn96z7_i2H7doGSNzMZp8sg-1UQeqeexk55lbk8HocZ-vwonbPtRF' +
-	'w8-Yr34RduS_oLJzthOjijjfboGVgMGEiZe8Lij9sGt8OkS690yehokto_U4Gn6My' +
-	'Q_aSLsqXj7NPMDdAv_7VX_HKsuTJEd7_pcT1sNTqtGxwCLsRIYTeLsyepd2yeWxmhhQrLKz3_cm4';
+var TOKEN = require('./tokens').client1;
+var DIFFERENT_CLIENT_ID_TOKEN = require('./tokens').client2;
```
Comment (azadorozhniy) [inline test/client.spec.js]: :+1:

**Comment by azadorozhniy** [comment]
Should we add specific test to test if we can handle multiply connections?

**Comment by mr-mig** [comment]
Merged manually

**Pull request closed without merging**